app/controllers/api/conversion_hosts_controller.rb

```diff
@@ -48,6 +48,10 @@ def create_resource(type, id, data)
 
       raise BadRequestError, "unsupported resource_type #{resource_type}" unless resource.supports_conversion_host?
 
+      conversion_host = ConversionHost.where(:resource => resource)
+
+      raise BadRequestError, "the resource '#{resource.name}' is already configured as a conversion host" unless conversion_host.empty?
```

Comment:
This really feels like it should be a validation on the model.

Reply:
Like this? https://github.com/ManageIQ/manageiq/pull/19945

Reply:
:+1: Thanks!